src/main/java/erlyberly/PrefBind.java

```diff
@@ -35,31 +36,58 @@ public static void bind(final String propName, StringProperty stringProp) {
 		if(props == null) {
 			return;
 		}
-		
 		String storedValue = props.getProperty(propName);
-		
 		if(storedValue != null) {
 			stringProp.set(storedValue);
 		}
-
 		stringProp.addListener(new ChangeListener<String>() {
 			@Override
 			public void changed(ObservableValue<? extends String> o, String oldValue, String newValue) {
 				props.setProperty(propName, newValue);
+				//TODO: simplify storing...
+				store();
+//				timer.schedule(new TimerTask() {
+//					@Override
+//					public void run() {
+//						if(!awaitingStore) {
+//							Platform.runLater(PrefBind::store);
+//							awaitingStore = true;
+//						}
+//					}}, 1000);
 				
-				timer.schedule(new TimerTask() {
-					@Override
-					public void run() {
-						if(!awaitingStore) {
-							Platform.runLater(PrefBind::store);
-							awaitingStore = true;
-						}
-					}}, 1000);
```

Comment:
This delayed write mechanism exists so that properties that change very can still be bound to the preferences. For example we could bind the width of table columns or split panes. When the user drags a table column the width property updates rapidly and each update would cause a file write to happen on the UX thread which would be bad.